package/MDAnalysis/analysis/align.py

```diff
@@ -332,6 +384,9 @@ def alignto(mobile, reference, select="all", mass_weighted=False,
        Uses :func:`get_matching_atoms` to work with incomplete selections
        and new *strict* keyword. The new default is to be lenient whereas
        the old behavior was the equivalent of *strict* = ``True``.
+
+    .. versionchanged:: 0.15.1
```

Comment:
@richardjgowers How should we handle things like this in the release when we decide to update the version number further then originally anticipated.